templates/robotics/ros-esm.html

```diff
@@ -44,251 +39,290 @@ <h4>
   </div>
 </section>
 
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>
-        What is ROS ESM?
-      </h2>
-      <p>
-        As your code develops and packages change, ROS needs to be continuously patched and updated, along with your other software dependencies. This time-consuming endeavour detracts from your robotics development. Duplicating and freezing upstream repositories is not a secure solution. Running un-patched and unmaintained versions of ROS exposes your robot, company and customers to serious risk. The challenge increases when working with End-Of-Life (EOL) ROS distributions or dependencies such as ROS Kinetic and Python 2.
-      </p>
-      <p>
-        Delivered in partnership with Open Robotics, ROS ESM provides a hardened and long-term supported ROS environment for your robot and its applications. With ROS ESM, you can count on backports for critical security updates, common vulnerabilities and exposures (CVE) fixes and bug fixes for ROS and Ubuntu base OS distribution.
-      </p>
-      <p>
-        <a href="/blog/what-is-ros-esm">
-          Learn more about ROS ESM&nbsp;&rsaquo;
-        </a>
-      </p>
+<section class="p-strip is-x-deep u-no-padding--bottom">
+  <div class="p-strip u-no-padding--top">
+    <div class="row">
+      <hr class="is-fixed-width">
+      <div class="col-6 col-medium-3">
+        <h2>What is ROS ESM?</h2>
+      </div>
+      <div class="col-6 col-medium-3">
+        <p>Whether you want to be compliant with cybersecurity regulations, win more time before your migration, or reduce operational expenses of maintaining upstream repositories, ROS ESM is here to make your work easier.</p>
```

Comment:
```suggestion
        <p>Whether you want to be compliant with cybersecurity regulations, win more time before your migration, or reduce operational expenses, ROS ESM is here to make your work easier.</p>
```
This bit was removed in the copydoc